src/main/java/seedu/address/logic/Messages.java

```diff
@@ -42,13 +42,13 @@ public static String format(Patient patient) {
                 .append(patient.getPreferredName())
                 .append("; PatientHospitalId: ")
                 .append(patient.getPatientHospitalId())
-                .append("; FoodPreference: ")
-                .append(patient.getFoodPreference())
-                .append("; FamilyCondition: ")
-                .append(patient.getFamilyCondition())
-                .append("; Hobby: ")
-                .append(patient.getHobby())
-                .append("; Tags: ");
+                .append("; FoodPreferences: ");
+        patient.getFoodPreferences().forEach(builder::append);
+        builder.append("; FamilyConditions: ");
+        patient.getFamilyConditions().forEach(builder::append);
+        builder.append("; Hobbies: ");
+        patient.getHobbies().forEach(builder::append);
+        builder.append("; Tags: ");
```

Comment:
Suggest that you could append a comma between each food preference, family condition and hobbies.
Currently the success message for edit patient lumps all the food preference together without spacing or commas. same for hobbies and family conditions.

As you can see from the screenshot, after editing a patient's family condition using `edit 8 f/1 f/2 f/3` the elements of each set is being lumped together in the feedback.

<img width="937" alt="image" src="https://github.com/AY2324S2-CS2103-F09-2/tp/assets/60460444/07ab21a5-104b-4943-be93-2c063cf26a8c">

Reply:
great suggestion!